nucypher/cli/characters/stake.py

```diff
@@ -678,56 +626,40 @@ def collect_reward(click_config,
 
 
 @stake.command('preallocation')
```

Comment:
This was a previous change done by @cygnusv , but this line is not needed since the function name is the same as the command.

Reply:
Thanks, I'll remove it.

Reply:
Removed in the new revision.